src/libcore/fmt/mod.rs

```diff
@@ -272,6 +270,14 @@ impl<'a> ArgumentV1<'a> {
     #[doc(hidden)]
     #[unstable(feature = "fmt_internals", reason = "internal to format_args!", issue = "none")]
     pub fn new<'b, T>(x: &'b T, f: fn(&T, &mut Formatter<'_>) -> Result) -> ArgumentV1<'b> {
+        // SAFETY: `mem::transmute(x)` is safe because
+        //     1. `&'b T` is transmuted to have an explicit lifetime `'b`
```

Comment:
`x` keeps the lifetime it originated with, `'b`, I think is more accurate -- transmuting to a different explicit lifetime would *not* be sufficient.

Reply:
I fixed the comment.